lib/config/presets/internal/helpers.ts

```diff
@@ -22,7 +22,7 @@ export const presets: Record<string, Preset> = {
     extends: [':followTag(typescript, rc)'],
   },
   pinGitHubActionDigests: {
-    description: 'Pin `github-action` digests.',
+    description: 'Pin `github-action` digests.\n In case you use `vX` pins and you\'d like not only pin digests but specify which exact tag is used by changing `vX` to `vX.Y.Z` - except enabling this preset, add to your config [this workaround](https://github.com/renovatebot/renovate/discussions/21901#discussioncomment-6563510).',
```

Comment:
### What I think of the current method

I don't like putting workaround information in the `description` for `helpers:pinGitHubActionDigests`. I think only _some_ readers will want this information.

Linking to a discussion makes the reader navigate away from the Renovate docs, into GitHub. They have to read the discussion, copy/paste the Renovate config code.

A smaller problem is that the Renovate code in the Discussion won't be updated whenever Renovate itself makes breaking changes to the configuration syntax. We generally update the Renovate config in our own presets when we make breaking changes.

### How about adding a new `workaround:*` preset?

I think we should add a `workaround:convertPinnedGitHubActionDigestToSemver` preset. Your preset would `extend` from `helpers:pinGitHubActionDigests`. I'm thinking of something like this:

```json
{
  "extends": ["helpers:pinGitHubActionDigests"],
  "packageRules": [
    {
      "description": "`v` prefix workaround for GitHub Action updates.",
      "extractVersion": "^(?<version>v\\d+\\.\\d+\\.\\d+)$",
      "versioning": "regex:^v(?<major>\\d+)(\\.(?<minor>\\d+)\\.(?<patch>\\d+))?$"
    }
  ]
}
```

Questions for the maintainer, @rarkins:

- What's the reason you wanted a "docs PR"?
- Would you accept a "new preset PR" instead?

Reply:
I expected a new preset PR instead